packages/vue-components/src/questions/Question.vue

```diff
@@ -396,16 +395,13 @@ export default {
     }
 
     @keyframes shake {
-        20% { transform: translate(-1px, -1px); }
-        40% { transform: translate(1px, 1px); }
-        60% { transform: translate(0, 0); }
-        80% { transform: translate(-1px, 1px); }
-        100% { transform: translate(1px, -1px); }
+        50% { transform: translate(-2px, 0); }
+        100% { transform: translate(2px, 0); }
```

Comment:
> Large animations in quiz should be avoided.

Just in case, I actually meant the sliding animation from question to question. 

But if this felt like an annoyance as well I'm ok with changing this too.

> sliding animation from question to question. 

Does this look OK? Personally, it was a little disorienting. I'm fine with keeping it as well otherwise.

Reply:
If you have any other ideas you'd like to add in, please do! You can get input on the design from the others here.

Reply:
I initially felt that the sliding animation was not too disorienting, however after taking another look I think there may be better alternatives we can explore. Especially for users who may want to complete a quiz quickly, I feel that this animation may be a tad too long. I will explore alternatives and create a shorter animation and push it in the next commit!

Reply:
In the latest commit I reduced the translateX to make the transition less distracting. However, I felt that using translateY would make for a nicer effect but doing this would block the Quiz's progress bar for a split second. Do you think that would be an issue?

Reply:
> reduced the translateX

This looks much better!

> translateY

Do you have a gif or similar? Just imagining it, I don't think it'd be an issue.

Reply:
![ezgif com-optimize](https://user-images.githubusercontent.com/71922282/217854039-31193c49-ce2c-48cf-b202-bc6e33bf2192.gif)
Here it is! There seems to be a split second of the progress bar being blocked by the incoming question as it comes from the top.

Reply:
That looks even better than horizontal transitions.

>  split second of the progress bar being blocked by the incoming question as it comes from the top.

Is that not fixable with z-index, or did doing so create a different UX?

Reply:
Fyi: transforms create their own stacking context if its the issue https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context

Reply:
I managed to fix the covering of the progress bar! I realised I was changing the z-index in the wrong field. I have updated the transition to translateY in the latest commit!

Reply:
Thanks, the transition looks good.

The z-index implementation can be improved slightly. If `1000` is larger than the zindex of other elements in the root/parent stacking context(s), it will appear above them. E.g. 

(I changed 1000 to 100000 to demonstrate easily here)
![image](https://user-images.githubusercontent.com/3306138/218233979-a7795b79-a179-40e6-9a8a-a402a78f0181.png)

Let's create our own stacking context to isolate the z-index's impact within the `<quiz>`.

We could likely use a smaller, more controlled value (e.g. just `z-index: 1`) for easier tracking then as well. (optional)

Reply:
Thank you for your suggestion! I agree as well and have isolated the z-index's impact within `<quiz>` and changed the `z-index` to `1` in the latest commit!